R.swift/Generators/StringsGenerator.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  StringsGenerator.swift
+//  R.swift
+//
+//  Created by Nolan Warner on 2016/02/23.
+//  Copyright © 2016 Nolan Warner. All rights reserved.
+//
+
+import Foundation
+
+struct StringsGenerator: Generator {
+  let externalStruct: Struct?
+  let internalStruct: Struct?
+
+  init(strings: Strings?) {
+    externalStruct = Struct(
+      type: Type(module: .Host, name: "string"),
+      implements: [],
+      typealiasses: [],
+      properties: strings?.localizedKeys.keys.map {
+        Let(isStatic: true, name: $0, typeDefinition: .Inferred(Type.Strings), value: "_R.string.localizedString(\"\($0)\")")
+      } ?? [],
+      functions: [],
+      structs: []
+    )
+
+    internalStruct = Struct(
+      type: Type(module: .Host, name: "string"),
+      implements: [],
+      typealiasses: [],
+      properties: [],
+      functions: [StringsGenerator.localizedStringFunction()],
+      structs: []
+    )
+  }
+
+  private static func localizedStringFunction() -> Function {
+    return Function(
```

Comment:
What is the goal of this function?

Why not use `NSLocalizedString` directly in the generated R.strings struct?